File: src/ui/asq-ui/src/app/_models/domain/host-summary-dto.ts
import { BaseDomainDto } from './base-domain-dto';

export class HostSummaryDto extends BaseDomainDto{
    public username: string = '';
    public name: string = '';
    public surname: string = '';
    public thumbnailUrl: string = '';
    public foci: string[] = [];

    constructor(){ super(); }
}

File: src/lib/models/DTO/Domain/HostSummaryDto.cs
using System.Collections.Generic;

namespace DTO.Domain
{
    public class HostSummaryDto: BaseDomainDto
    {
        public string Username{ get; set; }
        public string Name{ get; set; }
        public string Surname{ get; set; }
        public string Summary{ get; set; }
        public string ThumbnailUrl{ get; set; }
        //public IEnumerable<string> Foci{ get; set; }
    }
}
